sys/clif/clif.c

```diff
@@ -149,6 +152,14 @@ ssize_t clif_add_target(const char *target, char *buf, size_t maxlen)
     return pos;
 }
 
+ssize_t clif_add_target(const char *target, char *buf, size_t maxlen)
+{
+    assert(target);
+
+    size_t target_len = strlen(target);
```

Comment:
Looking at this now, not sure if a size `0` is a valid target?

Reply:
No it should not be a valid target. I added an assert.